### ISSUE TITLE
[PDP] When publishing changes that include a delete, receive error in console

### ISSUE THREAD

**Issue by zenweasel** [reactioncommerce/reaction#1484]
### Expected behavior

No error in console
### Actual Behavior

Receive this error in the browser console: (operation seems to be successful otherwise)

```
TypeError: Cannot read property 'title' of null
    at composer (socialContainer.js:29)
    at with_tracker.js:19
    at Tracker.Computation._compute (tracker.js:311)
    at Tracker.Computation._recompute (tracker.js:330)
    at Object.Tracker._runFlush (tracker.js:495)
    at onGlobalMessage (setimmediate.js:102)
```
### Steps to Reproduce the Behavior
1. Login as administrator
2. Make changes to product
3. Select "Delete Product" or Trashcan icon
4. Select "Publish Changes"
5. Observe error in console.

**Comment by zenweasel** [comment]
No longer applicable.

**Comment by zenweasel** [comment]
Spoke too soon. Didn't publish "archive" action. Failure still occurs.

**Comment by zenweasel** [comment]
Closed via #1619
